### PR TITLE
Fixes error handling when server rejects connection

### PR DESCRIPTION
**Pull request by luisremis**

### CODE REVIEW

File: aperturedb/BBoxLoader.py [hunk 1]
```diff
@@ -140,6 +140,7 @@ def generate_batch(self, bbox_data):
             fi = {
                 "FindImage": {
                     "_ref": img_ref,
+                    "blobs": False,
```
Comment (luisremis) [inline aperturedb/BBoxLoader.py:143]: this is another unrelated bug fix, where boundingbox loader is unnecessarily retrieving img blobs.

**Review by gsaluja9 — APPROVED** [review]
LG
